<launch>
    <param name="id" type="int" value = "0"/>
    <param name="preprocessed_map" type="bool" value = "true"/>
    <param name="map_name" type="str" value = "outside_ny_updated"/>
    <param name="map_extent" type="yaml" value = "[0,0,0,0]"/>
    <param name="desired_speed" type="double" value = "5"/>

    <param name="predefined_mission" type="bool" value = "false"/>
    <param name="predefined_mission_name" type="str" value = "stavanger_sauda_ryfylke"/>
    <param name="goal_pose" type="yaml" value = "[-73.843602,40.641420,0]"/>
    <param name="initial_pose" type="yaml" value = "[-73.991544,40.504594,0]"/>

    <param name="simulation_collection_name" type="str" value = "ny_robustness1_"/>
    <param name="simulations" type="int" value = "1"/>

    <param name="sim_origin" type="yaml" value = "[-73.991544,40.504594,0]"/>

    <include file="$(find usv_mission_planner)/launch/mission_planner.launch"/>
    <include file="$(find usv_map)/launch/default.launch"/>

    <rosparam command="load" file="$(find usv_simulator)/config/monte_carlo_supervisor/leader.yaml" />
    <node pkg="usv_simulator"
        name="monte_carlo_supervisor"
        type="monte_carlo_supervisor_node"
        respawn="false"
        output="screen"
        required="true">
    </node>

    <node pkg="usv_simulator"
            name="sim"
            type="sim_vessel_node"
            respawn="false"
            output="screen">
    </node>

</launch>